app/bundles/CampaignBundle/Event/EventPreview.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Mautic\CampaignBundle\Event;
+
+use Mautic\CampaignBundle\Entity\Event;
+
+class EventPreview
```

Comment:
Every new class should be final by default.
```suggestion
final class EventPreview
```